### DISCUSSION TITLE
Could not build wheels for amuse-athena

### DISCUSSION THREAD

**Discussion by BOOM5s** [amusecode/amuse#971] [Q&A] [answered]
i have installed amuse and package is not install because of this error
ERROR: Could not build wheels for amuse-athena, which is required to install pyproject.toml-based projects.
below i have attached the whole script.
please Help
[amuse.txt](https://github.com/amusecode/amuse/files/11994775/amuse.txt)

**Comment by rieder** [comment, marked as answer]
Hi BOOM5s (sorry, you didn't mention your real name),
I see a few possible issues, but I can't say for sure which is causing this problem.
A few things to get started solving this:
- It looks like you are using the XCode compilers (rather than e.g. Macports or homebrew), is that correct? If so, could you try with Macports? XCode on its own won't work for AMUSE.
- It also looks like you're installing AMUSE system-wide, this is really not recommended for a variety of reasons. Please could you try creating a virtualenv and installing inside this environment?

Hope this helps, 
Steven

**Reply by rieder** [reply]
Please see https://amuse.readthedocs.io/en/latest/install/howto-install-AMUSE.html for instructions for Macports and creating a virtualenv.

**Reply by rieder** [reply]
If this doesn't help, could you please post your AMUSE config.mk file? You can find it in the directory listed with this command:
```
amusifier --get-amuse-dir
```

**Reply by BOOM5s** [reply]
Sir I'm using homebrew as mention in the instruction 
I also created the amuse env but right now it's not getting activated 
yesterday i used amusifier command it show something different but today it gave me this error
himalayagaur@Himalayas-Air ~ % amusifier --get-amuse-dir
zsh: /Library/Frameworks/Python.framework/Versions/3.9/bin/amusifier: bad interpreter: /usr/local/bin/python3.9: no such file or directory
now I'm installing it by using tarball but i'm getting an error
[Untitled erroe.txt](https://github.com/amusecode/amuse/files/12000763/Untitled.erroe.txt)

**Reply by rieder** [reply]
You will need to activate the AMUSE env first.
The error indicates that the Python version you used to install AMUSE is no longer existing, it looks like you have upgraded it.
The other error indicates that you have not installed the prerequisite Python packages, please could you do `pip install -r requirements.txt` before re-running `configure`?

**Reply by BOOM5s** [reply]
but sir it is showing in my echo $PATH
himalayagaur@Himalayas-Air amuse-2023.5.0 % echo $PATH
/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/himalayagaur/opt/anaconda3/bin:/Library/Frameworks/Python.framework/Versions/3.9/bin:/var/root/usr/local/bin/rbin.042:/Users/himalayagaur/usr/local/bin/rbin.042:/Users/himalayagaur/usr/local/bin/rbin.042:/Users/himalayagaur/usr/local/bin/rbin.026:/usr/local/bin/rbin.026:/usr/local/bin/rbin.042/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
when i activate virtual environment and then I find in that environment I am using python 3.11 version

himalayagaur@Himalayas-Air ~ % source Amuse-ev/bin/activate
(Amuse-ev) himalayagaur@Himalayas-Air ~ % python3 --version
Python 3.11.4
also i have downloaded all the prerequiste python packages.
and then again start configue in virtual environment but make is still giving error.I have attached a file below
[Untitled12345.txt](https://github.com/amusecode/amuse/files/12024237/Untitled12345.txt)

here is my config.mk file
[confg.mk.txt](https://github.com/amusecode/amuse/files/12026666/confg.mk.txt)

here is error script when change Xcode clang compiler to GCC homebrew compiler, setting path for python 3.11.4,but it is giving error even now i tried via tarball as well pip install amuse. uploaded script is from pip install amuse
[pipinstallamuse.txt](https://github.com/amusecode/amuse/files/12026815/pipinstallamuse.txt)

installed packages in amuse environment
[installedpackageinamuse_env.txt](https://github.com/amusecode/amuse/files/12063790/installedpackageinamuse_env.txt)

installed packages systemwide macOS
[installedpackageinmac.txt](https://github.com/amusecode/amuse/files/12063791/installedpackageinmac.txt)
